pkgs/servers/passopolis/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, antBuild, fetchgit, git, python }:
+
+let
+  version = "b827b3a6176e050deb729009676fad7e86e5393a";
+in antBuild {
+  name = "passopolis-${version}";
```

Comment:
We usually use `passopolis-unstable-<year>-<month>-<day>` here as it is more meaningful then a commit hash (where the date is the date of the commit).